tests-beta/android/MASVS-CODE/MASTG-TEST-0290.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Enforced Application Updates
+platform: android
+id: MASTG-TEST-0290
+type: [static]
+weakness: MASWE-0036
+profiles: [L1, L2]
+---
+
+## Overview
+
+Enforced updates on Android occur when an application requires users to install the latest version before they can continue using it. This behavior is commonly implemented via the Google Play Core In-App Update API using the Immediate update type with a value `1`. When the Immediate update flow is triggered, the application blocks access to its functionality until the update is fully downloaded and installed.
```

Comment:
This overview rather describes the problem. This would mostly be covered by the weakness and not repeated here. It would be great to slightly rewrite it with the tester in focus. E.g. what's the goal of the test?

See MASTG-TEST-0207 for reference